ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -29,11 +29,11 @@
           </v-btn>
         </v-col>
         <v-col style="padding-top: 2px;">
-          <p v-if="isDraft(item)" :class="{ 'ma-0': true, 'pl-9': isChild}">Pending</p>
+          <p v-if="isDraft(item)" :class="{ 'ma-0': true, 'pl-9': isChild }">Pending</p>
           <!-- child drafts will sometimes show outside their base reg during the sort -->
           <div v-if="isChild || (isDraft(item) && item.baseRegistrationNumber)" :class="isChild ? 'pl-9' : ''">
-            <p v-if="!isPpr" style="font-size: 0.875rem !important;" :class="{ 'ma-0': true }">{{ item.documentId }}</p>
             <p v-if="isPpr" class="ma-0">{{ item.registrationNumber }}</p>
+            <p v-else style="font-size: 0.875rem;" :class="{ 'ma-0': true }">{{ item.documentId }}</p>
```

Comment:
This class binding is still set to always `true`. 

Using class binding, ie `:class={}` is a way to conditionally add a class. In this case it's always `true` so there is no condition. 
Was the intention here to apply a condition? 
If not, you can simply add the class to the element, ie `class="ma-0"`

Reply:
I am not sure why it is there... I did'nt change the existing code other than the condition... but I will make the change.

Reply:
Fair enough, all good! 
Thank you 👍